server/methods/accounts/accounts.js

```diff
@@ -912,10 +914,19 @@ export function setUserPermissions(userId, permissions, group) {
 function getEmailLogo(shop) {
   let emailLogo;
   if (Array.isArray(shop.brandAssets)) {
+    let mediaId;
```

Comment:
the changes from here on add extra null-checking/default value setting.
I changed them to make tests less verbose (which is typically is a bad reason for changing code!), but also figured extra null checks are never a bad thing.